lib/jose-utils.js

```diff
@@ -367,5 +367,15 @@ Utils.sha256 = function(str) {
 Utils.isCryptoKey = function(rsa_key) {
   // Some browsers don't expose the CryptoKey as an object, so we need to check
   // the constructor's name.
-  return rsa_key.constructor.name == 'CryptoKey';
+  if (rsa_key.constructor.name == 'CryptoKey') {
```

Comment:
Yes, that's what I was thinking. It might make it harder to test/develop across browsers but I feel it's safer to honor the built-in CryptoKey when it's defined.